tests/integration/netapi/rest_cherrypy/test_app.py

```diff
@@ -190,7 +193,7 @@ def test_accepts_arg_kwarg_keys(self):
                 'Accept': 'application/json',
             }
         )
-        resp = json.loads(response.body[0])
+        resp = json.loads(salt.utils.to_str(response.body[0]))
```

Comment:
Get the encoding from the headers and use that. to_str will use the filesystem encoding which is not what we need here.